settings.py

```diff
@@ -9,3 +9,6 @@
 INTEGRATION_TEST_CLIENT_SECRET = os.getenv("INTEGRATION_TEST_CLIENT_SECRET")
 
 PLUGIN_RUNNER_DAL_TARGET = os.getenv("PLUGIN_RUNNER_DAL_TARGET")
+
+DAL_GRAPHQL_ENDPOINT = os.getenv("DAL_GRAPHQL_ENDPOINT")
+DAL_GRAPHQL_AUTH_KEY = os.getenv("DAL_GRAPHQL_AUTH_KEY")
```

Comment:
I'm just being pedantic at this point, but...

```suggestion
GRAPHQL_ENDPOINT = os.getenv("DAL_GRAPHQL_ENDPOINT")
GRAPHQL_AUTH_KEY = os.getenv("DAL_GRAPHQL_AUTH_KEY")
```

Reply:
Good call, especially since we're retiring the DAL moniker.

Reply:
Done.